### PR TITLE
rootless: rename auth-scripts to preexec-hooks

### PR DESCRIPTION
**Pull request by giuseppe**
to not give a false sense of security since these are not a security mechanism but a hook to run arbitrary code before executing a command.

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
@baude PTAL

File: test/system/950-auth-scripts.bats
Comment (edsantiago) [inline test/system/950-auth-scripts.bats:32]: Would you consider renaming the file, updating the header comments and test name, and renaming the `$auth_dir` variable? Otherwise future maintainers are going to be really lost.

**Review by vrothberg — APPROVED** [review]
/lgtm
/hold

**Pull request merged**
